src/app/Models/tipoCliente.model.ts

```diff
@@ -0,0 +1,17 @@
+export interface TipoCliente {
+    idTipoCliente: Number;
+    descripcion: string;
+    activo: Number;
+}
+
+export class TipoClienteModel implements TipoCliente{
+    idTipoCliente: Number;
+    descripcion: string;
+    activo: Number;
+
+    constructor(item:any) {

```

Comment:
falta validar que los item venga por ejemplo this.idTipoCliente = item.idTipoCliente ?? this.idTipoCliente el doble singno de interrogacion dice si es nulo pone el siguiente valor